orga/app/routes/application.js

```diff
@@ -24,6 +27,26 @@ export default class ApplicationRoute extends Route.extend(ApplicationRouteMixin
     this._clearStateAndRedirect(redirectionUrl);
   }
 
+  async _getUserAndLocale(lang = null) {
+    const currentUser = await this._loadCurrentUser();
+    if (lang && this.currentUser.prescriber) {
+      this.currentUser.prescriber.lang = lang;
+      await this.currentUser.prescriber.save({ adapterOptions: { lang } });
+    }
+    await this._setLocale(lang);
+    return currentUser;
+  }
+
+  _setLocale(lang = null) {
+    const defaultLocale = 'fr';
+    let locale = lang || defaultLocale;
+    if (this.currentUser.prescriber) {
+      locale = this.currentUser.prescriber.lang || defaultLocale;
+    }
+    this.intl.setLocale([locale, defaultLocale]);
+    this.moment.setLocale(locale);
+  }
```

Comment:
let locale = 'fr';
if (this.currentUser.prescriber && this.currentUser.prescriber.lang) {
      locale = this.currentUser.prescriber.lang;
    }